.plzconfig

```diff
@@ -1,5 +1,5 @@
 [please]
-version = >=16.0.0-prerelease.14
+version = >=16.0.1
```

Comment:
haha much more civilised!

Reply:
Indeed :) There was some kind of bug in go_module which was the broken builds...